modules/indexer/stats/indexer.go

```diff
@@ -40,15 +40,18 @@ func populateRepoIndexer() {
 
 	isShutdown := graceful.GetManager().IsShutdown()
 
-	exist, err := db.IsTableNotEmpty("repository")
+	engine := db.GetEngine(db.DefaultContext)
+	tableRepository := new(models.Repository)
+
+	exist, err := engine.Table(tableRepository).Exist()
```

Comment:
No DB code in modules.

Reply:
The code was there. The code was accessing database directly via db.Xxx.

The new code is just more clear than before.

Adding new packages is not the task or purpose of this PR. This PR just makes a clean up.